lib/PackageMeta.js

```diff
@@ -1,14 +1,18 @@
 const Controller = require("./Controller");
 
-class PackageMeta extends Controller {
+class PackageMeta {
```

Comment:
it's better to not extend controller as it has no links with the package meta object.